.coveragerc

```diff
@@ -26,3 +26,5 @@ exclude_lines =
     .*:.* # Python \d.*
     .* # Abstract
     .* # Defensive:
+    if TYPE_CHECKING:
+    \.\.\.
```

Comment:
These allow no-coverage for:

* blocks that only evaluate during type checking, since they'll never execute at runtime
* the bodies of protocol stubs and overload stubs (`Ellipsis` is roughly equivalent to `pass` but is typically used in contexts where it doesn't actually ever execute directly)

Reply:
Actually considering whether I need to adjust the `\.\.\.` regex to be more defensive about only excluding coverage on lines where it's the only item, since it can be used in function type signatures (e.g., a function that takes `Callable[..., None]` as a param)

Reply:
Validated issue. This code was marked as coverage-not-required with `\.\.\.`:

```python3
def has_no_coverage(something: Callable[..., None]) -> None:
    return something()
```

Replacing regex with:
```regex
^\s*?\.\.\.\s*$
```

This checks that the only non-whitespace content in the whole line is an ellipsis, as when used for the contents of a class or function body.